scene/gui/scroll_container.cpp

```diff
@@ -238,32 +238,25 @@ void ScrollContainer::_update_scrollbar_position() {
 	_updating_scrollbars = false;
 }
 
-void ScrollContainer::_ensure_focused_visible(Control *p_control) {
-	if (!follow_focus) {
-		return;
+void ScrollContainer::_gui_focus_changed(Control *p_control) {
+	if (is_a_parent_of(p_control) && follow_focus) {
```

Comment:
This is really minor, but this would be better swapped (i.e. check `follow_focus` first). There is no reason to test parenting if focusing is disabled and the latter is cheaper.